File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
       int a=0;
       sort(nums.begin(), nums.end());
       for(int i=0;i<nums.size();i++){
           if(nums[i]==a){
               a++;
           }
       }
       return a;
    }
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = len(nums)
        xor = 0

        for num in nums:
            xor ^= num
        
        # missing = 0
        for i in range(0, n+1):
            xor ^= i
        return xor